frontend/pages/create-edit-user.js

```diff
@@ -0,0 +1,367 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Another note for the refactor/cleanup task, we should break out component, functions and constants from this file into smaller files we import into this one to reduce the size of this file

Reply:
Completely agree :)